### DISCUSSION TITLE
Broken Github Education for Teachers Organization Upgrade and Verified Sync Issue

### DISCUSSION THREAD

**Discussion by KrisJordan** [community/community#64945] [GitHub Education]
### Hi everyone,

I am a verified educator with GitHub last reverified Jul 26, 2022 and cannot re-request updated verification.

On Sunday 8/20, I requested the Teams upgrade benefit on an organization for a course that started this week (org: comp423-23f).

On Wednesday, 8/23, I received the ❤️ email from GitHub Education saying that the organization plan was updated. However, the organization is still on Free, rather than Teams.

This Fall, I am leading a Software Engineering Course for the second time; previously we were very successful in using Teams features through this upgrade. Any help on this front would be highly appreciated!

One other curious observation, in my Github Classroom associated with this organization, there is the pop-up on the assignments page that said "Get verified to get Codespaces Education Benefit". This seems out of sync as when I click the link it shows I've been verified for over a year and cannot request reverification.

Thanks!

**Comment by arafajs** [comment]
First, let's check the status of your organization's plan. Go to your GitHub Global Campus dashboard and click on the "Upgrade your academic organizations" section. If your organization is still on the Free plan, then the GitHub Education team may have not yet processed your request.

If it has been more than a few days since you received the email from GitHub Education, you can try contacting them directly. You can do this by opening a support ticket at https://github.com/contact.

When you contact GitHub Education, be sure to include the following information:

- Your organization's name
- The email address that you used to request the upgrade
- The date that you requested the upgrade
- A screenshot of the email from GitHub Education that says that your organization plan was updated
